### ISSUE TITLE
Certain METOC Symbols Not Renderering

### ISSUE THREAD

**Issue by david-melendez** [missioncommand/old-mil-sym-java#40]
I have pulled the latest renderer code and I am having a problem generating some METOC symbols.

- Fisk Stakes (_WOS-HPFS---L---_)
- Ferry Crossing (_WOS-HPM-FC-L---_)
- Cable Ferry Crossing (_WOS-HPM-CC-L---_)
- Offshore Loading Facility (_WO-DHPMO---L---_)

I believe the problem is that the xml that describes these symbols has been modified.  The geometry for these symbols have been changed from **_point_** to **_line_**.   I think that there is a disconnect between what the symbol ID indicates the geometry should be (all the symbols have "-L-" for graphic type which indicates a line) and the description of the required anchor points for the symbol (standard indicates that these are static symbols that require a _single_ anchor point).

Basically, I am trying to render the symbol by passing a single point for the graphic, but the renderer is looking for a minimum of two points.

**Comment by mdeutch3** [comment]
I saw the symbol for ferry crossing listed in the SIDC table but they don't show the graphic anywhere in table C-III.

**Comment by david-melendez** [comment]
The ferry crossing graphic is right before the cable ferry crossing in the C-III table (page 902 in the MIL-STD-2525C, 17 NOVEMBER 2008 document).

 

 

David Melendez

 <mailto:david.melendez@nova-tech-solutions.com> david.melendez@nova-tech-solutions.com

Software Engineer

Nova Tech Solutions

**Comment by mdeutch3** [comment]
Yes, they are both single point graphics.

**Comment by michael-spinelli** [comment]
I checked with the 2525D draft.  They should be single points.  Although D uses numerical symbol IDs.  Not sure if I should change the L to a P or just leave it as L.

**Comment by david-melendez** [comment]
The issue is not necessarily with the ID, although that is confusing.  The issue is that that the _SymbolConstantsB.xml/SymbolConstantsC.xml_ files have been changed to indicate that these symbols are not lines and not points.  Attempting to render these symbols with a single point returns an error that the symbols must have at least two points.  If we agree that these are single point files, the xml needs to be changed back to what it was before.

Here is a screenshot from the history of the xml file.

![image](https://cloud.githubusercontent.com/assets/22623059/22028775/faca490c-dc9d-11e6-8c96-fb343f71998b.png)

**Comment by michael-spinelli** [comment]
Going to leave it as 'L' for now, I'll switch those graphics back

**Comment by michael-spinelli** [comment]
fixed in v0.1.18
